### PR TITLE
refactor: remove building through npm scripts

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Includes:
* Comply docs with new way of webpacking
* Remove npm run script support 

Part of https://github.com/NativeScript/NativeScript/issues/5129

Ping @PanayotCankov @sis0k0 

### Breaking Changes
Executing `npm run` scripts is no longer supported. The scripts will print an error message and exit with error code 1.

### CODE REVIEW

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
Ping @sis0k0 with renewed PR message and topic

**Comment by sis0k0** [comment]
run ci

**Comment by vchimev** [comment]
typescript-app-android

**Comment by vchimev** [comment]
http://nsbuild01.telerik.com/build/job/tns-webpack-typescript-app-android-PR/47/console
```
Snapshot tools path: /var/folders/0z/36pc0x513611h_zzwnddp9zw3wctsh/T/snapshot-tools
Webpack compilation complete. Watching for file changes.
TypeError [ERR_INVALID_ARG_TYPE]: The "listener" argument must be of type Function
    at _addListener (events.js:180:11)
    at WriteStream.addListener (events.js:240:10)
    at WriteStream.close (fs.js:2302:10)
    at WriteStream.<anonymous> (/Users/nsbuilduser/workspace/tns-webpack-typescript-app-android-PR/nativescript-dev-webpack/demo/TypeScriptApp/node_modules/nativescript-dev-webpack/snapshot/android/utils.js:23:30)
    at WriteStream.emit (events.js:164:20)
    at finishMaybe (_stream_writable.js:605:14)
    at afterWrite (_stream_writable.js:456:3)
    at onwrite (_stream_writable.js:446:7)
    at fs.js:2246:5
    at FSReqWrap.wrapper [as oncomplete] (fs.js:703:5)
Cannot read property 'message' of undefined

```

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ A package to help with webpacking NativeScript apps.
 $ npm install --save-dev nativescript-dev-webpack
 $ npm install
```
Comment (MartoYankov) [inline README.md]: I believe this is no longer needed since tns run calls npm install.

**Pull request merged**